src/mailadm/mailcow.py

```diff
@@ -0,0 +1,56 @@
+import requests as r
+
+
+class MailcowConnection:
+    """Class to manage requests to the mailcow instance.
+
+    :param config: the mailadm config
+    """
+    def __init__(self, config):
+        self.config = config
+        self.auth = {"X-API-Key": config.mailcow_token}
+
+    def add_user_mailcow(self, addr, password, quota=1024):
+        """HTTP Request to add a user to the mailcow instance.
+
+        :param addr: the email address of the new account
+        :param password: the password  of the new account
+        :param quota: the maximum mailbox storage in MB
+        """
+        url = self.config.mailcow_endpoint + "add/mailbox"
+        payload = {
+            "local_part": addr.split("@")[0],
+            "domain": addr.split("@")[1],
+            "quota": quota,
+            "password": password,
+            "password2": password,
+            "active": True,
+            "force_pw_update": False,
+            "tls_enforce_in": True,
+            "tls_enforce_out": True,
+        }
+        result = r.post(url, json=payload, headers=self.auth)
+        if type(result.json()) != list or result.json()[0].get("type") != "success":
+            raise MailcowError(result.json())
+
+    def del_user_mailcow(self, addr):
+        """HTTP Request to delete a user from the mailcow instance.
+
+        :param addr: the email account to be deleted
+        """
+        url = self.config.mailcow_endpoint + "delete/mailbox"
+        result = r.post(url, json=[addr], headers=self.auth)
+        if type(result.json()) != list or result.json()[0].get("type") != "success":
+            raise MailcowError(result.json())
+
+#   def get_users(self):
+#       """HTTP Request to get all mailcow users (not only mailadm-generated ones)."""
+#       url = self.config.mailcow_endpoint + "get/mailbox/all"
+#       result = r.get(url, headers=self.auth)
+#       if result.json()[0]["type"] != "success":
+#           raise MailcowError(result.json())
+#       return result
```

Comment:
i'd add this method (taking into account also the json-comments above) and then create tests in a `test_mailcow.py` file testing basic add/del/get_users tests against a mailcow instance without all the other cmdline/db/conn machinery.  to make sure that the python wrappers against mailcow REST api work properly by themselves.

Reply:
I striked it :)

Reply:
I was a bit hesitant to add a method which is only used by tests, but if you see no problem with this, great :) and now we also use this function in conn.py:add_email_account, so...